src/main/java/mate/academy/bookshop/model/Book.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.bookshop.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
`@Data` is not recommended for use along with the `@Entity`, because of possible poor performance